runhouse/resources/module.py

```diff
@@ -220,11 +199,49 @@ def env(self):
     def env(self, new_env: Optional[Union[str, Env]]):
         self._env = _get_env_from(new_env)
 
+    @property
+    def signature(self):
+        return self._signature or {
+            name: self.method_signature(method)
+            for (name, method) in inspect.getmembers(self.__class__)
+            if not name[0] == "_" and name not in LOCAL_METHODS
+        }
+
+    def method_signature(self, method, rich=False):
+        """Extracts the properties of a method that we want to preserve when sending the method over the wire."""
+        if not callable(method):
+            return {
+                "signature": None,
+                "property": True,
+                "async": False,
+                "gen": False,
+                "local": False,
+            }
+
+        signature = inspect.signature(method)
+        ppts = {
+            "signature": str(signature),
+            "property": not callable(method),
```

Comment:
can set this to `False` to be clearer

Reply:
Not sure I understand, isn't that what this will do?

Reply:
yeah but I think including logic, aka like `not callable` when we know it is `False` is a bit cleaner, it's just a nit, either way is fine